### ISSUE TITLE
Remove startup logo screen or replace this screen with higher resolution image.

### ISSUE THREAD

**Issue by stevenleeS0ht** [subliun/Antox#517]
Due to the rapid increment of mobile phone hardware performance. The data loading speed is get faster and faster.

So the time for startup logo screen to appear would be extremely short.

The resolution of current logo screen is a little bit low. It would be influence the user experience.

I would like to recommend, that it would be awesome to remove this logo screen.

Or if it is very important to show the logo. I would be also awesome to show a SVG simple logo which is same as the App entry icon with pure color background.

**Comment by stevenleeS0ht** [comment]
@markwinter , what's your opinion?

**Comment by FuzzySock** [comment]
Isn't the startup logo just a rip-off of the Global Game Jam logo?

**Comment by life777eternal** [comment]
> Isn't the startup logo just a rip-off of the Global Game Jam logo?

Yeah now that you mention it, it is very similar. A modified version with the Tox lock image overlaid. Was wondering why Antox has Globe/Heliocentric propaganda.

**Comment by zoff99** [comment]
@stevenleeS0ht Antox has not been in active development for years.
please look at the current app:
https://f-droid.org/packages/com.zoffcc.applications.trifa/
https://github.com/zoff99/ToxAndroidRefImpl/tree/zoff99/dev003
